### ISSUE TITLE
Problem when running  run_ithemal.py

### ISSUE THREAD

**Issue by teoaivalis** [ithemal/Ithemal#65]
I am trying to train the model with bhive dataset and I take the following result.
![Screenshot from 2023-01-22 18-03-48](https://user-images.githubusercontent.com/68184383/213926215-daef5c0c-f931-43a6-80e4-539a29247991.png)

That's how I save the tuples on a .pkl file
![Screenshot from 2023-01-22 18-06-56](https://user-images.githubusercontent.com/68184383/213926245-3e816126-d6be-41c5-853a-918810ffab04.png)

Has anybody the same problem and succeed to solve it??

**Comment by alexrenda** [comment]
I've had that issue before when using different versions of software in the different parts of the system. Could you send both the code you're using to save the tuples (just the actual save is fine), along with the python version you're using for Ithemal and for saving tuples?

**Comment by teoaivalis** [comment]
In order to save the tuples I use Python 3.8.5 and save_data_bdw.py is the
file I run .
To train Ithemal I use docker so the python version there is Python 2.7.16
Could a solution be to use 2.7.16 and on my own computer to save the
tuples??

**Comment by alexrenda** [comment]
Yes, using Python 2.7 on your computer to save the tuples would likely fix the issue. You may also be able to use Python 3.8 with pickle protocol version 2 (https://docs.python.org/3/library/pickle.html), though I haven't tested this myself.

**Comment by teoaivalis** [comment]
Thanks maybe that was the problem 
now the training started!
